README.md

```diff
@@ -413,6 +413,21 @@ If it becomes necessary to exempt a namespace from Gatekeeper entirely (e.g. you
 
 > NOTE: Verbose logging with DEBUG level can be turned on with `--log-level=DEBUG`.  By default, the `--log-level` flag is set to minimum log level `INFO`. Acceptable values for minimum log level are [`DEBUG`, `INFO`, `WARNING`, `ERROR`]. In production, this flag should not be set to `DEBUG`.
 
+
+### Enable Delete Operations
+Delete operations needs to be enable by adding "DELETE" to the list of operations that call the webhook in the
```

Comment:
Can you pls add `operations that call the validation.gatekeeper.sh webhook` since we have two webhooks.

Reply:
Please also add `For admission webhooks registered for DELETE operations, use Kubernetes v1.15.0+.` as you can see here: https://github.com/open-policy-agent/gatekeeper/blob/master/pkg/webhook/policy.go#L130-L138